examples/seismic/model.py

```diff
@@ -340,7 +340,7 @@ def critical_dt(self):
         #
         # The CFL condtion is then given by
         # dt < h / (sqrt(ndim) * max(vp)))
-        dt = .95*np.min(self.spacing) / (np.sqrt(3)*self.maxvp)
+        dt = .95*np.min(self.spacing) / (np.sqrt(3.)*self.maxvp)
```

Comment:
Looks liek this was changed by mistake at some point. Should be 
`.85*np.min(self.spacing) / (np.sqrt(self.dim)*self.maxvp)`

Reply:
Ah cool. Will fix. Ta.